templates/_helpers.tpl

```diff
@@ -142,22 +142,16 @@ Insert service host environment variables
 {{- define "georchestra.service-envs" -}}
 - name: ANALYTICS_HOST
   value: "{{ include "georchestra.fullname" . }}-analytics-svc"
-- name: ATLAS_HOST
-  value: "{{ include "georchestra.fullname" . }}-atlas-svc"
 - name: CAS_HOST
   value: "{{ include "georchestra.fullname" . }}-cas-svc"
 - name: CONSOLE_HOST
   value: "{{ include "georchestra.fullname" . }}-console-svc"
-- name: EXTRACTORAPP_HOST
-  value: "{{ include "georchestra.fullname" . }}-extractorapp-svc"
 - name: GEONETWORK_HOST
   value: "{{ include "georchestra.fullname" . }}-geonetwork-svc"
 - name: GEOSERVER_HOST
   value: "{{ include "georchestra.fullname" . }}-geoserver-svc"
 - name: HEADER_HOST
   value: "{{ include "georchestra.fullname" . }}-header-svc"
-- name: MAPFISHAPP_HOST
```

Comment:
I don't think it's a good idea to remove this env, for DGE we still use mapfishapp.

Reply:
if you remove these lines, both mapfishapp AND extractorapp will not work anymore on DGE when we will rebase to the latest version of the chart on DGE.

Reply:
need to check with @fvanderbiest if we still use these apps for DGE